src/KubeLibrary/KubeLibrary.py

```diff
@@ -257,7 +257,7 @@ def reload_config(self, kube_config=None, context=None, api_url=None, bearer_tok
         if not self.api_client:
             self.api_client = client.ApiClient(configuration=client.Configuration().get_default_copy())
         self._add_api('v1', client.CoreV1Api)
-        self._add_api('extensionsv1beta1', client.ExtensionsV1beta1Api)
```

Comment:
I just had to fix the kubernetes library version to avoid problems with extensionsv1beta1 is out in latest varsion. So my suggestion is let's do the change, keeping v0.6.2 latest supporting extensionsv1beta1. @Nilsty any thoughts?

Reply:
This works for me. I think it's fair to drop it.